src/components/dictionaryConcepts/components/ConceptNameRows.jsx

```diff
@@ -169,7 +164,7 @@ class ConceptNameRows extends Component {
             className="btn btn-danger concept-form-table-link"
             id="remove-name"
             type="button"
-            onClick={event => this.handleRemove(event, this.props.newRow.uuid)}
+            onClick={event => this.handleRemove(event, this.props.rowId)}
```

Comment:
I think you should always destructure ur props

Reply:
Thanks. I have made the correction.